### DISCUSSION TITLE
Add icon to SVG Renderer

### DISCUSSION THREAD

**Discussion by btmi** [Shane32/QRCoder#318] [Ideas]
It would be nice to be able to insert SVG graphics in the centre of SVG rendered QR Codes like you can with pixel-rendered QR codes...even if the space could be left empty for manipulation afterwards, that would be a useful option.

**Comment by codebude** [comment]
This was already discussed in issue #281 
A new, optional paramter `SvgLogo` was added to SvgQRCode-class and will be available from release 1.4.2. (Or right now, if you pick the CI builds from [Github packages](https://github.com/codebude/QRCoder/packages).)

**Comment by btmi** [comment]
This is brilliant.  I checked out the CI build, and FYI the logo area needs to be cleared before the SVG is drawn as it is not empty when the image is rendered (although it still works).  Not sure I should be raising issues for preview builds though.

**Reply by codebude** [reply]
Please raise an issue and point out what needs to be changed. I plan to release 1.4.2 soon and I would be happy to release it without bugs. (Especially when they are known upfront. ;-) )
